### ISSUE TITLE
Filter box

### ISSUE THREAD

**Issue by jordanh** [ParabolInc/action#843]
## Issue - Enhancement

Let's make a thing like this: 

![image](https://cloud.githubusercontent.com/assets/103000/24170844/95984376-0e58-11e7-9fa2-0754dcb2fc66.png)

Suggested opening syntax, multiple tokens are interpreted as an AND query:

- `#tag` filters a tag
- `string` a substring within card content

For starters, let's do the filtering on the client.

- **Estimated effort:** 15 points ([see CONTRIBUTING.md](https://github.com/ParabolInc/action/blob/master/CONTRIBUTING.md#points-and-sizes))

**Comment by mattkrick** [comment]
is `team` a label? Is `user` a label?
Seems like a user dashboard is just every project in the world, filtered by your userId.
Similarly, your team dash is every project in the world, filtered by teamId.

If that's the case, and we got rid of actions, then the only different between My Dashboard & Team Dashboard is the Agenda List & the header, and if the agenda list is usually hidden, then we've got the same darn view. Seems like this is a good time to do a redesign.
